handbook/engineering/index.md

```diff
@@ -82,6 +82,12 @@ Sourcegraph has a lot of repositories!
 - [Infrastructure repositories](https://github.com/sourcegraph?utf8=%E2%9C%93&q=repo-type-infrastructure&type=&language=)
 - [Customer infrastructure repositories](https://github.com/sourcegraph?utf8=%E2%9C%93&q=repo-type-infrastructure+repo-type-customer&type=&language=)
 
+## Where Sourcegraph is deployed:
+
+- [Sourcegraph Cloud](https://sourcegraph.com/search)
+- [Sourcegraph About](https://about.sourcegraph.com)
```

Comment:
@valerybugakov we can probably remove this as it doesn't directly relate to engineering